src/index.ts

```diff
@@ -15,11 +15,12 @@
  * This also includes utility functions for operating on GraphQL types and
  * GraphQL documents to facilitate building tools.
  *
- * You may also import from each sub-directory directly. For example, the
- * following two import statements are equivalent:
- *
+ * You may also import from each sub-directory directly.
+ * @example the following two import statements are equivalent:
+ * ```ts
  *     import { parse } from 'graphql';
```

Comment:
Please convert it consistent indentation, because right now you have no spaces, 2 spaces, and 4 spaces variants.
Let's use no spaces variant across all code samples